openslides_backend/action/motion/reset_recommendation.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.schema import schema_version
+from ..generics import UpdateAction
+from ..register import register_action
+
+reset_schema = {
```

Comment:
Since all schemas now are arrays, we can generalize this further. But this doesn't have to be done here, if you don't want to, I can also do it in its own PR

Reply:
Nevermind, just had a look at it again and you can just use the default update schema without any properties since this is the exact same schema you're using here.

Reply:
Okay, use the DefaultSchema instead.